trafaret/contrib/rfc_3339.py

```diff
@@ -23,3 +23,27 @@ def check_and_return(self, value):
             return parse(value)
         except (ValueError, TypeError) as e:
             self._failure(str(e))
+
+
+class Date(Trafaret):
+    """Class for support parsing dates in RFC3339 formats
+    via dateutil.parse helper
+    """
+    convertable = str_types + (date,)
+    value_type = date
+
+    def __init__(self, allow_blank=False):
+        self.allow_blank = allow_blank
+
+    def __repr__(self):
+        return "<Date(blank)>" if self.allow_blank else "<Date>"
+
+    def check_and_return(self, value):
+        if isinstance(value, datetime):
+            return value.date()
+        elif isinstance(value, date):
+            return value
+        try:
+            return parse(value).date()
+        except (ValueError, TypeError) as e:
```

Comment:
here can be overflowerror

Reply:
What value will trigger it?

Reply:
http://dateutil.readthedocs.io/en/stable/parser.html?highlight=overflow

OverflowError – Raised if the parsed date exceeds the largest valid C integer on your system.

Reply:
Yep! I'll fix it tomorrow.